src/Compilers/CSharp/Portable/Utilities/GenericExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis
+{
+  public enum Clusivity : int
```

Comment:
I think we should use a different name here.  I agree that the term clusivity has the appropriate meaning for this use case but I don't think it's a common enough word for a type name.  

Admittedly I had to look up this word :smile: